cmd/main.go

```diff
@@ -95,21 +106,31 @@ func main() {
 		os.Exit(1)
 	}
 
-	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
-
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:             scheme,
-		MetricsBindAddress: metricsAddr,
-		LeaderElection:     enableLeaderElection,
-		LeaderElectionID:   "lahc7eig.amazee.io",
-		Port:               9443,
+		Scheme:                 scheme,
+		MetricsBindAddress:     metricsAddr,
+		Port:                   9443,
+		HealthProbeBindAddress: probeAddr,
+		LeaderElection:         enableLeaderElection,
+		LeaderElectionID:       "5153988a.amazee.io",
```

Comment:
Let's keep the ID for leader election the same:

```suggestion
		LeaderElectionID:       "lahc7eig.amazee.io",
```

Reply:
Same as above. More information [here](https://book.kubebuilder.io/migration/legacy/v2vsv3.html)

Reply:
I don't think we have to change the leader election ID. This could cause some weird issues during rollout because it would probably result in two leaders.